client_server/maxValues/maxAmpsVoltsParser.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Created by julia on 01/12/2020.
+//
+
+#include "maxAmpsVoltsParser.h"
+
+void parseMaxAmpsVolts(json_object_element_s* element, void * data) {
+    std::map <std::string, MaxAmpsVolts> * testResults = (std::map <std::string, MaxAmpsVolts> *) data;
+    MaxAmpsVolts maxValues;
+    const char * testName = element->name->string;
+    json_value_s * testValue = element->value;
+    struct json_object_s* object = json_value_as_object(testValue);
+
+    //TODO:Add check
+    struct json_object_element_s* maxValuesElement = object->start;
+
+    while (maxValuesElement != nullptr) {
+        const char *elementNameString = maxValuesElement->name->string;
+        json_value_s * elementValue = maxValuesElement->value;
```

Comment:
Code style: please use consistent `*` placement in this file.
https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions